app/controllers/publishers/vacancies_controller.rb

```diff
@@ -25,7 +25,7 @@ def save_and_finish_later
   end
 
   def create
-    vacancy = Vacancy.create(publisher: current_publisher, publisher_organisation: current_organisation)
+    vacancy = Vacancy.create(publisher: current_publisher, publisher_organisation: current_organisation, organisations: [current_organisation])
```

Comment:
We now need to have an organisation present on vacancy creation. The user can change organisations on the first page in the job creation flow as seen below.

![Screenshot 2024-05-29 at 16 30 25](https://github.com/DFE-Digital/teaching-vacancies/assets/13124899/ede42c0a-60cd-42ea-adc2-a4c02123996c)